Doc/library/sqlite3.rst

```diff
@@ -640,6 +640,11 @@ Cursor Objects
       .. versionchanged:: 3.6
          Added support for the ``REPLACE`` statement.
 
+   .. attribute:: arraysize
+
+      This attribute allows you to set the number of rows returned by :meth:`fetchmany`.
+      The default value is 1 meaning a single row would be fetched per call.
```

Comment:
Thank you.  You're right, I had used the word 'you' because of the other text in the sqlite3 documentation.  Overall, this one seemed much more conversational that the other docs I've read.  I added 'read/write' because I wanted to stress that the user is the one that sets the value.  I realize that may be obvious to most people, but they had to spell it out for me in the bpo.  :-(